### PR TITLE
core: remove remainings of the webworker platform.

### PR DESCRIPTION
**Pull request by JeanMeche**
See individual commits.

### CODE REVIEW

File: aio/tests/deployment/shared/URLS_TO_REDIRECT.txt [hunk 1]
```diff
@@ -124,8 +124,6 @@
 /docs/js/latest/api/platform-browser/index/Title-class.html --> /api/platform-browser/Title
 /docs/js/latest/api/platform-server/index/PlatformState-class --> /api/platform-server/PlatformState
 /docs/js/latest/api/platform-server/index/PlatformState-class.html --> /api/platform-server/PlatformState
-/docs/js/latest/api/platform-webworker --> /api/platform-webworker
```
Comment (JeanMeche) [inline aio/tests/deployment/shared/URLS_TO_REDIRECT.txt:127]: fyi the base rule `/docs/js/latest/api/ --> /api` is enough for a deleted page.

File: aio/firebase.json [hunk 2]
```diff
@@ -168,7 +168,6 @@
       // URLs that use the old scheme before we moved the docs to the angular/angular repo
       {"type": 301, "source": "/docs/*/latest", "destination": "/docs"},
       {"type": 301, "source": "/docs/*/latest/api/", "destination": "/api"},
-      {"type": 301, "source": "/docs/*/latest/api/:package", "destination": "/api/:package"},
```
Comment (JeanMeche) [inline aio/firebase.json:171]: This rule was unused because it was only matched by `/docs/js/latest/api/platform-webworker`

**Review by pkozlowski-opensource — APPROVED** [review]
LGTM

Reviewed-for: fw-core 
Reviewed-for: fw-testing

File: packages/core/src/linker/element_ref.ts [hunk 3]
```diff
@@ -49,25 +49,14 @@ export function createElementRef(tNode: TNode, lView: LView): ElementRef {
...
    *    data-binding provided by Angular instead. Alternatively you can take a look at {@link
-   * Renderer2}
-   *    which provides API that can safely be used even when direct access to native elements is not
-   *    supported.
-   *   </p>
-   *   <p>
-   *    Relying on direct DOM access creates tight coupling between your application and rendering
-   *    layers which will make it impossible to separate the two and deploy your application into a
-   *    web worker.
+   * Renderer2} which provides API that can safely be used
```
Comment (devversion) [inline packages/core/src/linker/element_ref.ts:57]: non-native english speaker suggestion 😄 

```suggestion
   * Renderer2} which provides an API that can be safely used.
```

**Review by devversion — APPROVED** [review]
LGTM thx for the cleanup

**Review by hexiaokang — APPROVED** [review]

**Comment by alan-agius4** [comment]
Caretaker note: G3 failures are pre-existing. 

[Presubmit](http://fusion2/presubmit/527865932/OCL:527865932:BASE:527865957:1682689965342:d1a708be/targets)

**Review by jessicajaniuk — APPROVED** [review]

**Comment by pkozlowski-opensource** [comment]
This PR was merged into the repository by commit 2419c486ad8a3129d27ade6852e33d1435512da1.

**Comment by fortheday001** [comment]
If webwoker is removed, What are the alternatives to make angular run in the SharedWorker?
Thx.

**Comment by JeanMeche** [comment]
@fortheday001 You want to run angular is the sharedWorker of having some code run in the shared worker in your angular App ?

**Pull request closed without merging**